recaptcha_enterprise/demosite/src/main/java/app/MainController.java

```diff
@@ -140,7 +185,8 @@ public static ModelAndView signup() {
     String recaptchaAction = PROPERTIES.getProperty("recaptcha_action.signup");
     HashMap<String, HashMap<String, String>> data = new HashMap<>();
     Assessment assessmentResponse;
-    String verdict;
+    String reason = "";
```

Comment:
Nit: a lot of this looks copy and paste. Would it be best to refactor this logic into a single function?

Reply:
Thanks Averi. Agreed.
Initially it made sense to have it in individual methods because it was not a lot of code to encapsulate. But it totally makes sense now to have a separate method for the common parts.